### DISCUSSION TITLE
Mauritania country  (+222 ) is missing from SMS 2FA

### DISCUSSION THREAD

**Discussion by mbareck7** [community/community#69884] [New to GitHub] [answered]
Mauritania country  (+222 ) doesn't appear in the list when adding my number as 2FA (SMS) 

at : `https://github.com/settings/security?type=sms`

![Screenshot from 2023-10-12 07-51-03](https://github.com/community/community/assets/23238771/2b342e06-0269-44e3-96db-bac68a7adbd7)

**Comment by queenofcorgis** [comment, marked as answer]
If your country is not on [the list](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication), then we aren't currently able to reliably deliver text messages to your country. We track delivery rates across all countries, including ones not in the list, and review it to understand when we have to remove countries or can add them back. We do not add countries with low deliverability rates, because it leads to users being locked out of their accounts when they can’t receive the SMS for 2FA. 

If GitHub doesn’t support two-factor authentication via text message for your country of residence, you must set up authentication via a TOTP application which is actually preferred. For more information on how to configure 2FA, please review our [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app).

**Comment by mbareck7** [comment]
I can see, thank you.

**Comment by queenofcorgis** [comment]
We’re glad you could get this solved and we want to make sure others can do the same so we’ve reopened this discussion for better discoverability for our members.
